core/views.py

```diff
@@ -235,6 +235,10 @@ def _add_contacts_to_object(self, message):
             if contact not in self.obj.contacts.all():
                 self.obj.contacts.add(contact)
 
+            structure = contact.get_structure_contact()
+            if structure and structure not in self.obj.contacts.all():
+                self.obj.contacts.add(structure)
```

Comment:
la méthode `add()` n'ajoute pas de doublons, donc pas besoin de vérifier avant si la structure est déjà dans contacts
```suggestion
            if structure := contact.get_structure_contact():
                self.obj.contacts.add(structure)
```